test/performance/alignment/multiple_alignment_benchmark.cpp

```diff
@@ -0,0 +1,59 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/alignment/multiple/align_multiple.hpp>
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/test/performance/sequence_generator.hpp>
+
+
+static void arguments(benchmark::internal::Benchmark* b)
+{
+    for (int32_t i : {10, 50})     // i = sequence length.
+    {
+        for (int32_t n : {20, 50}) // n = number of sequences.
+        {
+            b->Args({n, i});
+        }
+    }
+}
+
+static void seqan3_msa_similar_sequences(benchmark::State & state)
+{
+    auto n = static_cast<size_t>(state.range(0));
+    assert(n > 1);
+    auto i = static_cast<size_t>(state.range(1));
+    assert(i > 0);
+    // generate set of similar sequences with (0.3 * sequence_length) mutations per sequence.
+    auto seqs = seqan3::test::generate_similar_sequence_set<seqan3::dna4>(i, n, 0.3);
+
+    for (auto _ : state)
+    {
+        benchmark::DoNotOptimize(seqan3::align_multiple(seqs));
+    }
+}
+
+static void seqan3_msa_random_sequences(benchmark::State & state)
+{
+    auto n = static_cast<size_t>(state.range(0));
+    assert(n > 1);
+    auto i = static_cast<size_t>(state.range(1));
+    assert(i > 0);
+    // generate random sequence set.
+    auto seqs = seqan3::test::generate_random_sequence_set<seqan3::dna4>(i, n, 1);
```

Comment:
Can you tell me why you chose `sequence_variance = 1` and what the implications are?

Reply:
No actually not. I think this is a relict from me playing around with these values. I should have reset it to zero as is the default of all the other seq generator function. I think the randomness of the bases is created by varying the seed. Varying the sequence variance makes the resulting sequences vary in length. I think we don't need that plus the gerate_similar_seqeunce_set() does not do it either. 
I'll set it back to zero 👍